### DISCUSSION TITLE
backup hfs

### DISCUSSION THREAD

**Discussion by tatobarresi** [rejetto/hfs#365] [Q&A]
Hi everyone, how do I make a full hfs backup?. I have everything configured on a server and would like to transfer everything to a new machine. How should I proceed with the backup so as not to break the folder tree?

**Comment by rejetto** [comment]
if you use hfs.exe, then backup the whole folder where it is in.
other distributions create a folder ".hfs", you backup that.

**Comment by rejetto** [comment]
i took the opportunity to add this new page on the wiki https://github.com/rejetto/hfs/wiki/Backup

**Reply by tatobarresi** [reply]
sto testando, vedo però che scompaiono alcune immagini che avevo impostato nella pre-login page

**Comment by rejetto** [comment]
e dove sono i file di queste immagini?

**Comment by tatobarresi** [comment]
in una cartella dentro la directory principale

**Reply by rejetto** [reply]
che intendi con cartella principale?

**Comment by tatobarresi** [comment]
dentro CARTELLA MADRE, ho la cartella "img" che contiene tutte le immagini per customizzare l'html oltre che tutte le cartelle dei dati condivisi con HFS
![Cattura](https://github.com/rejetto/hfs/assets/145445552/3a0cbf96-7c25-4a19-a5ed-e0105ba6a5ab)

**Comment by rejetto** [comment]
e "cartella madre" è negli "shared files" di hfs ?
forse è il caso che condividi config.yaml, altrimenti sono troppe le variabili in gioco
